internal/integrations/github/github_test.go

```diff
@@ -0,0 +1,52 @@
+package github
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/h2non/gock.v1"
+)
+
+func TestGetTravisCIBuildURLForRef_Success(t *testing.T) {
+	defer gock.Off()
+
+	mockResponse, _ := ioutil.ReadFile("testdata/check_runs_response_success.json")
+	gock.New("https://api.github.com").
```

Comment:
https://github.com/google/go-github didn't seem to have a way of stubbing a response at a higher level so this just intercepts the HTTP call to the GitHub Checks API and injects a response that I `curl`'d myself from the API.

Reply:
Another option would be to use an interface to decouple our `Github` struct type from the `go-github/github.Client`, and then mock out that dependency in these test cases.